patches/1.12/patch1/arcgis-experience-builder/readme.md

```diff
@@ -0,0 +1,9 @@
+This patch fixes typescript errors when you import the `esri/...` modules.
+
+Steps:
+
+1. [Download](https://github.com/Esri/arcgis-experience-builder-sdk-resources/archive/refs/heads/master.zip) the repository code and unzip it.
+2. Browse to the patch directory (`patches/1.12/patch1/arcgis-experience-builder`)
+3. Replace the files in the `client` folder with the files in the `client` folder on your Experience Builder installation.
+4. Run `npm ci` in the `client` folder
+5. Restart the node server in both `client` and `server` folders
```

Comment:
Are these steps in the instructions necessary? After I updated the `tsconfig.json` all I had to do was restart VS Code.

```suggestion
```